### DISCUSSION TITLE
Applicazione UNOCONV per la conversione in PDF

### DISCUSSION THREAD

**Discussion by nicolatrentini** [iisgiua/giuaschool#497] [Richieste di aiuto]
buonasera 
vi chiedo gentilmente di rispondermi a questo messaggio perchè mentre vado su https://nome-sito.altervista.org/public/install/app.php
le cose obbligatorie sono tutte installate tranne quella Opzionale della UNOCONV .
per cortesia mi aiuti Grazie ,

**Comment by trinko** [comment]
UNOCONV non è obbligatorio perché su molti servizi di hosting (come Altervista) non si può installare.
Sarebbe infatti necessario un servizio di hosting che ti permetta di installare applicazioni sul sistema operativo del server, cosa che solo certi servizi a pagamento permettono di fare.
Puoi comunque andare avanti nell'installazione senza problemi.

**Comment by nicolatrentini** [comment]
Ok

Il gio 27 apr 2023, 00:21 Antonello Dessì ***@***.***> ha
scritto:

**Comment by trinko** [comment]
Aggiungo che, se dopo aver terminato l'installazione da **https://nome-sito.altervista.org/public/install/app.php** viene fuori un errore (o una pagina bianca), probabilmente significa che è necessario reimpostare la versione del PHP a 8.0 sul pannello di controllo Altervista.
